style/license-header

```diff
@@ -0,0 +1,17 @@
+/*
```

Comment:
We already use skywalking to check the license header, is this needed? this will only check the java code? or all files.

Reply:
I'm not sure whether `skywalking-eyes` could add the license header for users or not. `Spotless` could automatically add it if there is no license header.

Reply:
Skywalking-eyes will only check if the file contains license header, does Spotless will add license header in all files?
We need to filter some `md`, `json` file.

Reply:
Sure, I will check it to make sure `md` and `json` excluded.

Reply:
> I'm not sure whether `skywalking-eyes` could add the license header for users or not. `Spotless` could automatically add it if there is no license header.

License eye can add license headers and it deals with all files. I think spotless only deals with the files within maven project's files?
 Anyway I don't think we should have two tools to do the same thing

Reply:
Got it, will remove it in the next commit.